Doc/whatsnew/3.12.rst

```diff
@@ -510,7 +510,7 @@ Pending Removal in Python 3.14
 
   (Contributed by Jason R. Coombs and Hugo van Kemenade in :gh:`93963`.)
 
-* Creating :c:data:`immutable types <Py_TPFLAGS_IMMUTABLETYPE>` with mutable
+* Creating immutable types (:data:`Py_TPFLAGS_IMMUTABLETYPE`) with mutable
```

Comment:
Seems like the real problem is that this is documented as Python data, when it is in fact C data (a flag, specifically). So perhaps this should stay how it is, until the real problem is fixed (especially since doing so won't break external links, as the generated fragment _should_ be the same.